etc/config.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Store:etc/config.xsd">
+    <default>
+        <vendic_checkout_newsletter_subscribe>
+            <general>
+                <checked_by_default>1</checked_by_default>
```

Comment:
@kis28viktor 
Doesn't it make more sense to set this to 0? This module is used for multiple client and with setting it to 1 I believe the behaviour is changed for these clients when module is